packages/ccd-js-gen/README.md

```diff
@@ -559,3 +595,37 @@ const invokeResult = await MyContract.dryRunLaunchRocket(myContract, invoker, pa
 // Parse the error message
 const message: MyContract.ErrorMessageLaunchRocket | undefined = MyContract.parseErrorMessageLaunchRocket(invokeResult);
 ```
+
+#### function `create<EntrypointName>ParameterWebWallet`
+
+For each entrypoint of the smart contract a function for _constructing_ the WebWallet formattet parameter is produced. This is to be used with the [`@concordium/wallet-connector`](https://www.npmjs.com/package/@concordium/wallet-connectors) package.
+These are named `create<EntrypointName>ParameterWebWallet` where `<ContractName>` is the smart contract name in Pascal case.
+
+_This is only generated when the schema contains contract entrypoint parameter type._
+
+An example for a smart contract with an entrypoint named `launch-rocket`, the function
+becomes `createLaunchRocketParameterWebWallet`.
+
+**Parameter**
+
+The function parameter is:
+
+- `parameter` Parameter to provide to the smart contract entrypoint.
+
+**Returns**: Parameter for updating a contract instance in the format used by [`@concordium/wallet-connector`](https://www.npmjs.com/package/@concordium/wallet-connectors).
+
+```typescript
+// Wallet connection from `@concordium/wallet-connector`.
+const webWalletConnection: WalletConnection = ...;
+// Parameter to pass the smart contract init-function. The structure depends on the contract.
+const parameter: MyContract.LaunchRocketParameter = ...;
+// Convert the parameter into the format expected by the wallet.
+const walletParameter = MyContract.createLaunchRocketParameterWebWallet(parameter);
```

Comment:
I had to add a type assertion here. Might be useful to mention. The `ccd-js-gen` tool has no access to `TypedSmartContractParameters` type from the `@concordium/wallet-connector` package (since it is not a dependency of this project). So the types are not identical.

```
const walletParameter = MyContract.createLaunchRocketParameterWebWallet(parameter) as unknown as TypedSmartContractParameters;
```

Reply:
It should not complain even without access to the type directly, since Typescript is a structural typing system. So that (should) mean that something is actually different in the types... I'll look into this.

Reply:
The issue seems to be that the wallet does not support using `bigint` as part of smart contract parameters, the generated will use `bigint` for anything that cannot be represented by `number` in JS, so the generated types are currently incompatible.

Reply:
Is tracked in this issue https://github.com/Concordium/concordium-browser-wallet/issues/420